test/models/vote_test.rb

```diff
@@ -0,0 +1,15 @@
+require "test_helper"
+
+describe Vote do
+  it 'can be instantiated' do
+    (Vote.new).must_be_kind_of Vote
```

Comment:
better to use a fixture

```suggestion
    expect(votes(:new_vote)).must_be_kind_of Vote
```